src/modules/physics/physics_jolt.c

```diff
@@ -0,0 +1,1474 @@
+#include <stdlib.h>
+#include "core/maf.h"
+#include "util.h"
+#include "physics.h"
+#include "joltc.h"
+
+#define MAX_BODIES 2048
```

Comment:
This value is too small and only sufficient for testing.
In the official hello world, the recommended value for the production environment is around 65535.
MaxBodyPairs, MaxContactConstraints also need to be larger
https://github.com/jrouwe/JoltPhysics/blob/cc49e07d7c4eb854a60cb21ea0abdfcc93511410/HelloWorld/HelloWorld.cpp#L239

Reply:
Thanks for catching this, I've set the values to the ones in the linked document.